.ci/docker/conda-env-ci.txt

```diff
@@ -1,4 +1,5 @@
 cmake=3.22.1
 ninja=1.10.2
 libuv
+llvm-openmp
```

Comment:
Is this needed?  I thought that this has been added by this https://github.com/pytorch/executorch/blob/main/.ci/docker/common/install_clang.sh#L16